app/src/main/java/org/oppia/android/app/testing/AdministratorControlsFragmentTestActivity.kt

```diff
@@ -15,11 +18,14 @@ import org.oppia.android.app.testing.activity.TestActivity
 import org.oppia.android.app.translation.AppLanguageResourceHandler
 import javax.inject.Inject
 
-/** Activity [AdministratorControlsFragmentTestActivity] that allows user to change admin controls. */
+/** Test activity for ``AdministratorControlsFragmentTestActivity``. */
```

Comment:
```suggestion
/** Test activity for ``AdministratorControlsFragmentTest``. */
```

I think this is what you meant, or no? The previous documentation didn't really make sense since it was tautological.

Reply:
yeah thanks ben